React/CoreModules/RCTAlertController.h

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+#import <React/RCTUIKit.h> // TODO(macOS ISS#2323203)
+
+#if TARGET_OS_OSX // [TODO(macOS ISS#2323203)
+#define UIAlertController NSViewController
+#endif // ]TODO(macOS ISS#2323203)
+
+@interface RCTAlertController : UIAlertController
```

Comment:
@HeyImChris @thymikee : we are adding subclass of uialertcontroller when Apple documentation says "The UIAlertController class is intended to be used as-is and does not support subclassing. The view hierarchy for this class is private and must not be modified." ?

Reply:
https://developer.apple.com/documentation/uikit/uialertcontroller?language=objc

Reply:
Right, didn't notice that. It comes from the upstream https://github.com/facebook/react-native/commit/f319ff321c4b7c0929b99e3ebe7e1ce1fa50b34c